prow/plugins/trigger/pr.go

```diff
@@ -206,3 +215,12 @@ func buildAll(c client, pr github.PullRequest) error {
 	}
 	return nil
 }
+
+func hasLabel(str string, labels []github.Label) bool {
```

Comment:
If this PR makes sense, this helper can be replaced by the public helper in #5245. Wait until that PR merged.